src/channel_internal.c

```diff
@@ -337,10 +410,10 @@ IMPLEMENT_MOCKABLE_FUNCTION(, CHANNEL_RESULT, channel_internal_pull, THANDLE(CHA
         /*Codes_SRS_CHANNEL_INTERNAL_43_108: [ If the first operation in the list of pending operations contains a non-NULL push_callback: ]*/
         else if (CONTAINING_RECORD(channel_internal_ptr->op_list.Flink, CHANNEL_OP, anchor)->push_callback != NULL)
         {
-            if (dequeue_operation(channel_internal, out_op_pull, pull_callback, pull_context, NULL, NULL, NULL) != 0)
+            if (dequeue_operation(channel_internal, out_op_pull, correlation_id, pull_callback, pull_context, NULL, NULL, NULL, NULL) != 0)
             {
                 /*Codes_SRS_CHANNEL_INTERNAL_43_023: [ If there are any failures, channel_internal_pull shall fail and return CHANNEL_RESULT_ERROR. ]*/
-                LogError("Failure in dequeue_operation(channel_internal_ptr, out_op_pull, pull_callback, pull_context, NULL, NULL, NULL)");
+                LogError("Failure in dequeue_operation(channel_internal_ptr, out_op_pull, correlation_id, pull_callback, pull_context, NULL, NULL, NULL)");
```

Comment:
>LogError("Failure in dequeue_operation(channel_internal_ptr, out_op_pull, correlation_id, pull_callback, pull_context, NULL, NULL, NULL) [](http://example.com/codeflow?start=16&length=136)

Since the LogError was initially written, `dequeue` has gotten a new argument, but logging didn't keep up with it.

Log all arguments with their values.

Reply:
[https://msazure.visualstudio.com/One/_backlogs/backlog/Azure%20Messaging%20GeoReplication/Backlog%20items?workitem=29879341](https://msazure.visualstudio.com/One/_backlogs/backlog/Azure%20Messaging%20GeoReplication/Backlog%20items?workitem=29879341)